### DISCUSSION TITLE
CSV importer trying to open non existing page

### DISCUSSION THREAD

**Discussion by her0e** [firefly-iii/firefly-iii#5249] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Self hosting a docker CSV importer and Firefly 3 (5.6.2) on Synology NAS.

Set token for authentication.

But web shows this:

Firefly III CSV Import Tool
v2.6.1
Welcome! This tool will help you import CSV files into Firefly III. It works by reading your CSV file (possibly, according to a pre-generated configuration file) which will be submitted to the API endpoints of your Firefly III installation.

This tool is sparsely documented, you can find all the details you need in the documentation. Any links you see to the docs will open in a new window or tab.

You're using a Personal Access Token to authenticate to Firefly III.

**Error accessing "http://x.x.x.x:x//api/v1/about". Status code is 404. Body is: {"message":"Resource not found","exception":"NotFoundHttpException"}**

Seems to be trying to open non existing page, not sure why this happened.

### Expected behaviour

Expect to start importing file

### Debug information

do not know where to get those

**Comment by JC5** [comment]
This is not a bug but a configuration error. Did you follow the instructions in [.env.example](https://github.com/firefly-iii/csv-importer/blob/main/.env.example)?

**Comment by her0e** [comment]
:D

I did go through the documentation  on
https://docs.firefly-iii.org/csv/install/configure/  but not this exact
file. This sentence is missing  there...
2) Make sure you REMOVE any trailing slash from the end of the URL.

Thank you for your prompt response. And I apologise for wasting your
time unnecessarily.

**Reply by JC5** [reply]
No worries! I added the sentence 😁
